conf/dynaconf_hooks.py

```diff
@@ -0,0 +1,137 @@
+import json
+from pathlib import Path
+
+from robottelo.logging import logger
+from robottelo.utils.ohsnap import dogfood_repository
+from robottelo.utils.url import is_url
+
+
+def post(settings):
+    settings_cache_path = Path(f'settings_cache-{settings.server.version.release}.json')
+    if getattr(settings.robottelo.settings, 'get_fresh', True):
```

Comment:
Why `get_fresh`for repos is added to the robottelo settings and not to the `repos.yaml.template`. Afterall we are trying to fetch the fresh repos here.

So I believe `repos.yaml` is a perfect place for it !